.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+## Description
+
+_Describe the purpose of and changes within this Pull Request._
+
+## References
+
+_List links to any related tasks (GitHub Issues, Pull Requests, 3rd part task managers)_
+
+## Testing
+
+_How was this tested? If manually tested, list all steps necessary for the reviewer to confirm._
```

Comment:
I'd just go for a simple:

```
- [ ] Have tests been added/updated?
```

Requiring tests to be run manually should be a major exception going forward.